glslc/README.asciidoc

```diff
@@ -124,6 +135,10 @@ Shader Stage Selection>> table.
   extension, the file extension is ignored and the `-fshader-stage=` argument
   is used instead.
 
+CAUTION: `-fshader-stage=` overrides file extension; that means it should not
+be used together with SPIR-V assembly files because glslc will treat the given
+SPIR-V assembly files as of GLSL source code of the given shader stage.
```

Comment:
Explain that if it's necessary to use both assembly files and `-fshader-stage` on the same line, the assembly files should come first, before any `-fshader-stage`.